export/export.go

```diff
@@ -0,0 +1,275 @@
+package export
+
+import (
+	"context"
+	"runtime"
+
+	"golang.org/x/sync/errgroup"
+	"golang.org/x/sync/semaphore"
+	"golang.org/x/xerrors"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	v1 "k8s.io/client-go/applyconfigurations/core/v1"
+	confstoragev1 "k8s.io/client-go/applyconfigurations/storage/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	v1beta2config "k8s.io/kube-scheduler/config/v1beta2"
+)
+
+type Service struct {
+	client              clientset.Interface
+	podService          PodService
+	nodeService         NodeService
+	pvService           PersistentVolumeService
+	pvcService          PersistentVolumeClaimService
+	storageClassService StorageClassService
+	schedulerService    SchedulerService
+}
+
+// Resources  will used to compile all the resources for export.
+type Resources struct {
+	Pods            []corev1.Pod                              `json:"podList"`
```

Comment:
nits: `List` is not needed. (other resources as well)
```suggestion
	Pods            []corev1.Pod                              `json:"pods"`
```

Reply:
oops...I'd forgotten...